ui/panels-plugin/src/plugins/bar-chart/BarChart.ts

```diff
@@ -27,5 +27,6 @@ export const BarChart: PanelPlugin<BarChartOptions> = {
       content: BarChartOptionsEditorSettings,
     },
   ],
+  supportedQueryInterfaces: ['TimeSeriesQuery', 'TraceQuery'],
```

Comment:
(nit) I think query interfaces is good enough

Reply:
Ah yes indeed it's probably too long. I blindly followed the "specs" I had on that onehehe

If I would think twice, Inoticed in the code that sometimes we mention querytypes sometimes plugintypes. It's not yet clear to me.